### PR TITLE
STP-3102: Improvements to resiliency testing procedures

### PR DESCRIPTION
**Pull request by nrockershousen**
## Summary and Scope

The Resiliency_Testing_Procedure.md file has several issues that need to be addressed. Here are a couple for context:

- Acronyms are not properly capitalized
- Third party terms are not properly capitalized
- Redundant use of "NCN node"
- Large bulleted lists with the tasks buried in the text
- Spelling and formatting

## Issues and Related PRs

* Merge with [837](https://github.com/Cray-HPE/docs-csm/pull/837) + [836](https://github.com/Cray-HPE/docs-csm/pull/836)
* Resolves [STP-3102](https://jira-pro.its.hpecorp.net:8443/browse/STP-3102)

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

N/A

### CODE REVIEW

File: operations/resiliency/Resiliency_Testing_Procedure.md [hunk 1]
```diff
@@ -1,240 +1,386 @@
...
-   username =  "uastest"
-   uai_host =  "ncn-w001"
-   uai_status =  "Running: Ready"
-   uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
-   uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
-   uai_age =  "1m"
-   uai_name =  "uai-uastest-5653e9b9"
-   ```
-   Login to the created UAI (example):
+      ```
```
Comment (rgross727) [inline operations/resiliency/Resiliency_Testing_Procedure.md:216]: The output shown in this file is from a 1.3 system. Use the following for 1.5:
(You may want to replace the "vers" user name with something else?)

[[results]]
uai_age = "2m"
uai_connect_string = "ssh vers@10.103.8.170"
uai_host = "ncn-w002"
uai_img = "registry.local/cray/cray-uai-cos-2.1.70:latest"
uai_ip = "10.103.8.170"
uai_msg = ""
uai_name = "uai-vers-f8fa541f"
uai_status = "Running: Ready"
username = "vers"

File: operations/resiliency/Resiliency_Testing_Procedure.md [hunk 2]
```diff
@@ -1,240 +1,386 @@
...
-   ```bash
-   ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
-   ```
-   To verify the configuration of slurm, for example, within the UAI:
+   1. Log in to the created UAI.
+
+      For example:
+
+      ```bash
+      ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
```
Comment (rgross727) [inline operations/resiliency/Resiliency_Testing_Procedure.md:232]: Replace "ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
with:

ncn# ssh vers@10.103.8.170

File: operations/resiliency/Resiliency_Testing_Procedure.md [hunk 3]
```diff
@@ -1,240 +1,386 @@
...
+      ```
    
+   1. Verify the configuration of Slurm, for example, within the UAI:
+   
+      ```bash
+      uastest@uai-uastest-5653e9b9:/lus/uastest> srun -N 4 hostname | sort
```
Comment (rgross727) [inline operations/resiliency/Resiliency_Testing_Procedure.md:238]: Replace the "uastest@uai-uastest-5653e9b9:/lus/uastest> srun -N 4 hostname | sort"
with, to match the UAI above:

vers@uai-vers-f8fa541f-5c5c9f5b75-vsb2k:/lus> srun -N 4 hostname | sort

File: operations/resiliency/Resiliency_Testing_Procedure.md [hunk 4]
```diff
@@ -1,240 +1,386 @@
...
+1. Copy an MPI application source and WLM batch job files to the UAI.
+
+1. Compile an MPI application with the UAI. Launch application as batch job (not interactive) on compute node(s) that have not been designated, already, for reboots once an NCN is shut down.
+   
+   1. Verify that batch job is running and that application output is streaming to a file. Streaming output will be used to verify that the batch job is still running during resiliency testing. A batch job, when submitted, will designate a log file location. This log file can be accessed to be able to verify that the batch job is continuing to run after an NCN is brought down and once it is back online. Additionally, the `squeue` command can be used to verify that the job continues to run (for Slurm).
+   
+   1. Delete the UAI session when all of the testing is complete.
+   
+      ```bash
+      ncn# cray uas delete --uai-list uai-uastest-5653e9b9
```
Comment (rgross727) [inline operations/resiliency/Resiliency_Testing_Procedure.md:259]: Replace "ncn# cray uas delete --uai-list uai-uastest-5653e9b9"
with, to match the above UAI:

ncn# cray uas delete --uai-list uai-vers-f8fa541f

File: operations/resiliency/Resiliency_Testing_Procedure.md (hunk 1 above)
Comment (nrockershousen) [reply operations/resiliency/Resiliency_Testing_Procedure.md:216]: fixed

File: operations/resiliency/Resiliency_Testing_Procedure.md (hunk 2 above)
Comment (nrockershousen) [reply operations/resiliency/Resiliency_Testing_Procedure.md]: fixed

File: operations/resiliency/Resiliency_Testing_Procedure.md (hunk 3 above)
Comment (nrockershousen) [reply operations/resiliency/Resiliency_Testing_Procedure.md]: fixed

File: operations/resiliency/Resiliency_Testing_Procedure.md (hunk 4 above)
Comment (nrockershousen) [reply operations/resiliency/Resiliency_Testing_Procedure.md]: fixed

**Review by rgross727 — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**